### DISCUSSION TITLE
Atoms in lammps don't move

### DISCUSSION THREAD

**Discussion by psn417** [deepmodeling/deepmd-kit#190] [Q&A]
Hello, I get a .pb file from dpgen, an use it to run a lammps simulation, but the atoms don't move in every frame. This is trained by smooth version.
When I don't use smooth version, lammps run correctly.
Is it a bug or I make some mistakes?

this is my input file:

dimension 	3
boundary	p p p
units		metal
atom_style	atomic

read_data pos
mass * 23

pair_style deepmd graph.pb
pair_coeff

timestep	0.001

dump 2 all atom 10 2000.atom

fix         1 all nvt temp 1 2000 0.1
run         2500
unfix       1

fix	    2 all nvt temp 2000 2000 0.1
run         7500
unfix       2

S. Pan

**Comment by psn417** [comment]
dpgen gives me 4 .pb files, I should use them together or use a single one?

**Comment by njzjz** [comment]
Hi,
Can you dump forces in lammps and see whether forces are zero?


————————
Jinzhe Zeng
PhD student in Chemistry
Rutgers University

jinzhe.zeng@rutgers.edu
CV: https://njzjz.win/cv

**Comment by jameswind** [comment]
Also, can you report the training performance? Say, the training and
testing error, etc.

**Comment by psn417** [comment]
Thank you! I find that I must add "velocity create" command in lammps script to let the atoms move. It's strange that I can use lammps without this command when I don't use smooth version.
I also wonder which of the 4 .pb files should I use?

**Comment by njzjz** [comment]
This behavior is weird... Can you ensure forces are not zero? The velocity
command only gives an initial velocity.

The usage of 4 files is to print their standard deviations. You can see the
article of dpgen for details: https://doi.org/10.1016/j.cpc.2020.107206
--

Jinzhe Zeng
PhD student in Chemistry
Rutgers University

jinzhe.zeng@rutgers.edu
CV: https://njzjz.win/cv
